app/models/ebean/ReportExecution.java

```diff
@@ -208,5 +202,61 @@ public int hashCode() {
             return Objects.hash(reportId, scheduled);
         }
     }
+
+    /**
+     * A container for errors that occurred during report execution.
+     */
+    @JsonTypeInfo(
+            use = JsonTypeInfo.Id.NAME,
+            defaultImpl = ErrorString.class,
+            property = "type"
+    )
+    @JsonSubTypes({
+            @JsonSubTypes.Type(value = ErrorString.class, name = "ERROR_STRING"),
+    })
+    public abstract static class Error {
```

Comment:
Oh. No.

https://docs.oracle.com/javase/7/docs/api/java/lang/Error.html

This would be a major source of confusion based on the name.

Reply:
I thought about this too and I was going to name it something like `ExecutionError` but then I figured that would be redundant because this is an inner class.

As in, I didn't think it bought much to have a different name because it will already be used as `ReportExecution.Error`.

Reply:
Please add a comment to `ReportExecution.Errror` to rename the class.

Reply:
I'll just remove/revert it. There's not as obvious as a gain as I had thought from moving this to a class instead of a Map, and removing it avoids these problems entirely.